nsxt/resource_nsxt_nat_rule.go

```diff
@@ -245,41 +239,43 @@ func resourceNsxtNatRuleUpdate(d *schema.ResourceData, m interface{}) error {
 	//match_service := d.Get("match_service").(*NsServiceElement)
 	matchSourceNetwork := d.Get("match_source_network").(string)
 	natPass := d.Get("nat_pass").(bool)
+	firewallMatch := model.NatRule_FIREWALL_MATCH_MATCH_INTERNAL_ADDRESS
```

Comment:
do we know what version this was added in?

Reply:
nat_pass was removed at 4.0.0
firewall_match exists in 2.4 - it seemed old enough so I didn't check earlier versions...

Reply:
Maybe I should keep the code which uses the old API for pre-4.0.0 environments?

Reply:
I think current code should be good, I would run the test against 3.0.0 to be sure